### DISCUSSION TITLE
internlm2-20b-chat面对较长提示词，模型输出会错乱

### DISCUSSION THREAD

**Discussion by rabum** [InternLM/InternLM#699] [General]
我写了个教大模型对问题进行分类的提示词，internlm2-20b-chat输出分类后，开始重复输出毫无关联的一句话：
![image](https://github.com/InternLM/InternLM/assets/58140899/f5b2de28-00f9-49b4-9f36-b0ef4853555e)
请问有什么办法解决

**Comment by Willpower2662** [comment]
我用internlm2_5-7b-chat时也是这样，我的提示词也比较长，模型一直重复输出，直到最大值。
